src/main/java/com/java401/AlbumCookies401/controllers/HelloController.java

```diff
@@ -17,11 +17,11 @@
 public class HelloController {
 
     @Autowired
-    AlbumCookiesRepository albumCookiesRepository;
+    SongRepository albumCookiesRepository;
```

Comment:
If you're trying to retrieve albums, this needs to be an AlbumCookiesRepository. You can autowire both repositories into this controller if you need to.